app/_hub/kong-inc/acl/how-to/_examples.md

```diff
@@ -0,0 +1,92 @@
+---
+nav_title: ACL Examples
+title: ACL Examples
+---
+
+## ACL Configuration with Kong Gateway
+
+In this example, we will cover a common use case: As an API Owner, you want to regulate access based on the type of request methods and consumer groups. Specifically, our goal is to allow consumers in the dev group to perform GET, POST, and PUT requests on all routes, while reserving the DELETE request functionality exclusively for consumers in the admin group.
+
+
+### Create consumer groups 
+
+Using the API, create a consumer group for  `dev`
+```bash
+curl --request POST \
+    --url http://localhost:8001/consumer_groups \
+    --header 'Content-Type: application/json' \
+    --header 'accept: application/json' \
+    --data '{"name":"dev"}'
```

Comment:
Why are we passing attributes as json data for these examples? Normally we use the default content type and send each attribute separately. I can understand if the goal is to provide konnect instructions, but then the URL makes no sense.

I think we should either convert these examples to the standard format we use in the docs (eg `--data name=dev`), or have two tabs for each section: one for Gateway, and one for Konnect. Then this formatting makes sense.

Reply:
@jschmid1

Reply:
No specific reason, I'll update this PR with the `--data "key=value"` formatting.